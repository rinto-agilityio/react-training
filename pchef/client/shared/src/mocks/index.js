// mock categories
export const category = {
  name: 'Heathy',
  imgUrl:
    'https://images.pexels.com/photos/461198/pexels-photo-461198.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500',
}
