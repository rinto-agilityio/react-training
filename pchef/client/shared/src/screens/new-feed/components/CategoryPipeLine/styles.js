// Lib
import { StyleSheet } from 'react-native'

// Themes
import { METRICS, FONTS } from '../../../../themes'

const styles = StyleSheet.create({
  pipelineWrapper: {
    display: 'flex',
    flexDirection: 'row',
    padding: METRICS.largePadding,
  },
  pipelineItem: {
    marginRight: METRICS.largeMargin,
  },
  pipelineItemName: {
    fontWeight: FONTS.fontWeight.medium,
    textAlign: 'center',
    paddingBottom: METRICS.mediumPadding,
  },
})

export default styles
