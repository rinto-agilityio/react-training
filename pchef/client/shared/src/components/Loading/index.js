// Libs
import React from 'react'
import { View, ActivityIndicator } from 'react-native'

// Themes
import { COLORS } from '../../themes'

// Styles
import styles from './styles'

type Props = {
  size?: string,
}

const Loading = ({
  size = 'small',
}: Props) => (
  <View style={styles.wrapper}>
    <ActivityIndicator size={size === 'medium' ? 'small' : size} color={COLORS.baseGray} />
  </View>
)

Loading.defaultProps = {
  size: 'small',
}

export default Loading
