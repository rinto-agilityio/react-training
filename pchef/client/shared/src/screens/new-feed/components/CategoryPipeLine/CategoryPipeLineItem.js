import React from 'react';
import { View, Text, TouchableOpacity } from 'react-native'
import { Avatar } from 'react-native-paper'

// Styles
import styles from './styles'

type Props = {
  category: {
    id: string,
    name: string,
    imgUrl: string,
  },
  onPressCategoryPipeline?: (name: string) => void,
}

const CategoryPipeLineItem = ({
  category,
  onPressCategoryPipeline = () => {},
}: Props) => {
  const { id, name, imgUrl } = category

  return (
    <View style={styles.pipelineItem}>
      <Text style={styles.pipelineItemName}>{name}</Text>
      <TouchableOpacity onPress={() => onPressCategoryPipeline(id)}>
        <Avatar.Image
          style={styles.pipelineItemImage}
          size={70}
          source={{
            uri: imgUrl,
          }}
        />
      </TouchableOpacity>
    </View>
  )
}

CategoryPipeLineItem.defaultProps = {
  onPressCategoryPipeline: () => {},
}

export default CategoryPipeLineItem
