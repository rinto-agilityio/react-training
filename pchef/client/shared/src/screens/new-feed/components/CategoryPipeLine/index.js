// @flow
// add flow above to fix for using flow with React.memo

// Libs
import React, { memo } from 'react'
import { View } from 'react-native'

// Components
import CategoryPipeLineItem from './CategoryPipeLineItem'

// Styles
import styles from './styles'
import Loading from '../../../../components/Loading'

import type { FollowCategoryType } from '../../../../types'

type Props = {
  followCategory: Array<FollowCategoryType>,
  onPressCategoryPipeline?: () => void,
  loading: boolean,
}

const CategoryPipeLine = ({
  followCategory,
  onPressCategoryPipeline,
  loading,
}: Props) => (
  <View style={styles.pipelineWrapper}>
    {loading ? (
      <Loading />
    ) : (
      followCategory &&
      followCategory.map(item => (
        <CategoryPipeLineItem
          key={item.id}
          category={item}
          onPressCategoryPipeline={onPressCategoryPipeline}
        />
      ))
    )}
  </View>
)

CategoryPipeLine.defaultProps = {
  onPressCategoryPipeline: () => {},
}

export default memo<Props>(CategoryPipeLine)
