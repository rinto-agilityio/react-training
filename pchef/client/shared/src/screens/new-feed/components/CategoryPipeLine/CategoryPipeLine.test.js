// Components
import CategoryPipeLine from '.'
import Loading from '../../../../components/Loading'
import CategoryPipeLineItem from './CategoryPipeLineItem'

// Mocks
import { categories } from '../../../../mocks'

const props = {
  followCategory: categories,
  onPressCategoryPipeline: jest.fn(),
}

describe('Components', () => {
  describe('CategoryPipeLine', () => {
    let component

    beforeEach(() => {
      component = shallow(<CategoryPipeLine {...props} />)
    })

    it('Should render CategoryPipeLine component', () => {
      const renderComponent = renderer.create(<CategoryPipeLine {...props} />)
      expect(renderComponent).toMatchSnapshot()
    })

    it('Should render Loading component if is loading', () => {
      component.setProps({ loading: true })
      expect(component.find(Loading)).toHaveLength(1)
    })

    it('Should call onPressCategoryPipeline when press on category', () => {
      const category = component.find(CategoryPipeLineItem).at(0)
      category.props().onPressCategoryPipeline()
      expect(props.onPressCategoryPipeline).toHaveBeenCalled()
    })
  })
})
