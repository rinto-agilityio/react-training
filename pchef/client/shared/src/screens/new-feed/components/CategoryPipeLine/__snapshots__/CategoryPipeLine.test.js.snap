// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components CategoryPipeLine Should render CategoryPipeLine component 1`] = `
<div
  className="css-view-1dbjc4n r-display-6koalj r-flexDirection-18u37iz r-padding-1pcd2l5"
>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Chicken
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Heathy
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Easy Breakast
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Dessert
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Fish
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Pizza
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Cake
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Deserts
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Salad
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Kids
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Meat
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-marginRight-1mf7evn"
  >
    <div
      className="css-text-901oao r-fontWeight-majxgm r-paddingBottom-1mi0q7o r-textAlign-q4m81j"
      dir="auto"
    >
      Easy Lunch
    </div>
    <div
      className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundColor": "rgba(98,0,238,1.00)",
            "borderBottomLeftRadius": "35px",
            "borderBottomRightRadius": "35px",
            "borderTopLeftRadius": "35px",
            "borderTopRightRadius": "35px",
            "height": "70px",
            "width": "70px",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
          style={
            Object {
              "borderBottomLeftRadius": "35px",
              "borderBottomRightRadius": "35px",
              "borderTopLeftRadius": "35px",
              "borderTopRightRadius": "35px",
              "height": "70px",
              "width": "70px",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
            style={
              Object {
                "backgroundImage": "url(\\"https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg\\")",
              }
            }
          />
          <img
            alt=""
            className="css-accessibilityImage-9pa8cd"
            draggable={false}
            src="https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
